src/App.tsx

```diff
@@ -8,7 +7,47 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [selectType, setSelectType] = useState('All');
+  const [input, setInput] = useState('');
+  const [todoId, setTodoId] = useState(0);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await getTodos()
+        .then(response => {
+          setTodos(response);
+          setIsLoading(false);
+        });
```

Comment:
if you use async/await, don’t need to use then

```suggestion
      const data = await getTodos();
      
      setTodos(data);
      setIsLoading(false);
```